vcpkg/produce/export-compiled-dependencies.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Tutorial: Export compiled dependencies"
+description: In this tutorial we show you how to export compiled dependencies to be consumed by another project.
+author: JavierMatosD
+ms.author: Javiermat
+ms.date: 11/06/2023
+ms.prod: vcpkg
+ms.topic: tutorial
+# CustomerIntent: As an advanced vcpkg user I want to export compiled dependencies to be consumed by another project.
+---
+
+# Tutorial: Export compiled dependencies
+
+In this tutorial, we'll walk through how to export compiled dependencies using vcpkg. This can be particularly useful when you want to share libraries across multiple projects or for ease of distribution.
+
+## Prerequisites:
+
+- [Visual Studio](<https://visualstudio.microsoft.com/downloads/>) with C++ development workload and [CMake component](</cpp/build/cmake-projects-in-visual-studio#installation>)
+- [Git](<https://git-scm.com/downloads>)
+* vcpkg
+* A terminal
+
+## 1 - Install `fmt` library
+
+Begin by opening your terminal and navigating to the directory where vcpkg is installed. Once there, install the `fmt` library for your target platform:
+
+```console
+vcpkg install fmt:x64-windows
+```
+
+Replace `x64-windows` with your specific target triplet if different.
+
+## 2 - Export `fmt` library
+
+After installing `fmt`, you can export it by running:
+
+```console
+vcpkg export fmt:x64-windows --zip
+```
+
+You should see something similar to the following:
+
+```console
+The following packages are already built and will be exported:
+    fmt:x64-windows
+  * vcpkg-cmake:x64-windows
+  * vcpkg-cmake-config:x64-windows
```

Comment:
Aren't host dependencies useless in this case?
As far as I know they're only supposed to be used during the build of a port but here the port is already built.